pynuodb/connection.py

```diff
@@ -96,6 +98,9 @@ def __init__(self, dbName, broker, username, password, options):
             if 'cipher' in options and options['cipher'] == 'None':
                 self.__session.set_encryption(False)
 
+        parameters['clientProcessId'] = str(getpid())
+        parameters['clientHost'] = platform.node()
+
```

Comment:
Can you please add a test? It would especially be testing that the setting of connection properties works correctly. Also, could you write a test that sets the connection property clientInfo and make sure that works correctly? Thanks.